analyzers/src/SonarAnalyzer.CSharp/Rules/SymbolicExecution/SymbolicExecutionRunner.cs

```diff
@@ -112,11 +111,12 @@ protected override void Initialize(SonarAnalysisContext context)
 
         protected override void AnalyzeSonar(SyntaxNodeAnalysisContext context, bool isTestProject, bool isScannerRun, SyntaxNode body, ISymbol symbol)
         {
-            var enabledAnalyzers = SonarRules.Where(x => x.SupportedDiagnostics.Any(descriptor => IsEnabled(context, isTestProject, isScannerRun, descriptor))).ToArray();
-            if (!Configuration.ForceSonarCfg) // ToDo: Will be removed in RuleFactory PR
-            {
-                enabledAnalyzers = enabledAnalyzers.Where(x => !x.SupportedDiagnostics.Any(descriptor => AllRules.Keys.Any(dd => descriptor.Id == dd.Id))).ToArray();
-            }
+            var enabledAnalyzers = AllRules.Select(x => x.Value.CreateSonarFallback(Configuration))
+                                           .WhereNotNull()
+                                           .Cast<ISymbolicExecutionAnalyzer>() // ISymbolicExecutionAnalyzer should be passed as TSonarFallback to CreateFactory. Have you passed a Roslyn rule instead?
+                                           .Union(SonarRules)
+                                           .Where(x => x.SupportedDiagnostics.Any(descriptor => IsEnabled(context, isTestProject, isScannerRun, descriptor)))
+                                           .ToList();
```

Comment:
This should be indented 4 spaces according to coding style. Can be fix in any other PR